react/TaxTooltip.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FunctionComponent } from 'react'
+import { Tooltip, IconInfo } from 'vtex.styleguide'
+import TranslateTotalizer from 'vtex.totalizer-translator/TranslateTotalizer'
+
+import FormattedPrice from './components/FormattedPrice'
+
+const TaxTooltip: FunctionComponent<Props> = ({ taxesTotals }) => {
+  const tooltipLabel = (
+    <ul className="list pa0 ma0">
+      {taxesTotals.map(total => (
+        <li className="flex justify-between items-center" key={total.name}>
+          <span className="pr5">
+            <TranslateTotalizer totalizer={total} />
+          </span>
+          <span>
+            <FormattedPrice value={total.value} />
+          </span>
+        </li>
+      ))}
+    </ul>
+  )
+
+  return (
+    <Tooltip label={tooltipLabel}>
+      <span className="pointer flex items-start pl2 pt1">
+        <IconInfo size={12} />
+      </span>
+    </Tooltip>
+  )
+}
+
+interface Props {
+  taxesTotals: OrderItemTotal[]
+}
```

Comment:
```suggestion
interface Props {
  taxesTotals: OrderItemTotal[]
}

const TaxTooltip: FunctionComponent<Props> = ({ taxesTotals }) => {
  const tooltipLabel = (
    <ul className="list pa0 ma0">
      {taxesTotals.map(total => (
        <li className="flex justify-between items-center" key={total.name}>
          <span className="pr5">
            <TranslateTotalizer totalizer={total} />
          </span>
          <span>
            <FormattedPrice value={total.value} />
          </span>
        </li>
      ))}
    </ul>
  )

  return (
    <Tooltip label={tooltipLabel}>
      <span className="pointer flex items-start ml2 mt1">
        <IconInfo size={12} />
      </span>
    </Tooltip>
  )
}
```

Reply:
Moved the `props` interface up for keeping the standard used in the project 🤙

Reply:
(And changed from padding to margin which I think is semantically more correct in this case)